components/lib/calendar/Calendar.js

```diff
@@ -144,9 +144,10 @@ export const Calendar = React.memo(
 
         const updateValueOnInput = (event, rawValue, invalidCallback) => {
             try {
-                const value = parseValueFromString(rawValue);
+                const value = parseValueFromString(props.timeOnly ? rawValue.replace('_','') : rawValue);
```

Comment:
I noticed that timeOnly without mask work fine, this will pass for example '12:1' and 'isValidSelection' wont accept this as valid time, but with mask prop value will have underline at empty cels '12:1_' which can be approved by 'isValidSelection'

Reply:
We can modify code for accepting date like 12:1, however i guess this will have complex logic

Reply:
Yes it gets quite tricky real fast :)

Reply:
Replacing underline better add to 'parseDateTime' directly, and invoke replace method only if mask is passed